app/Http/Kernel.php

```diff
@@ -41,6 +41,7 @@ class Kernel extends HttpKernel
             'guard:api',
             'bindings',
             // 'throttle:60,1',
+            'cors' => \Rogue\Http\Middleware\Cors::class,
```

Comment:
also tried just `\Rogue\Http\Middleware\Cors::class,` here too and still throwing this error: 
```
Failed to load http://rogue.test/api/v3/signups?filter[campaign_id]=1283&orderBy=quantity%2Cdesc: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'null' is therefore not allowed access. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```